doc/ompl_interface/ompl_interface_tutorial.rst

```diff
@@ -71,3 +85,12 @@ The configuration of these optimization objectives can be done in the *ompl_plan
 
 For more information on the OMPL optimal planners, the reader is referred to the
 `OMPL - Optimal Planning documentation <http://ompl.kavrakilab.org/optimalPlanning.html>`_.
+
+Post-Processing Smoothing
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note: there is a limit to how much smoothing can help reduce indirect routes.
+
+You can adjust the amount of time MoveIt! spends on smoothing by increasing the planning time. Any remaining time after an initial plan is found, but before the ``allowed_planning_time`` is exhausted, will be used for smoothing.
+
+Although not currently exposed at the top levels of MoveIt! (TODO), more smoothing can be accomplished by setting the simplification duration to 0 (unlimited) in ``model_based_planning_context.cpp``. This will enable OMPL's ``simplifyMax()`` function.
```

Comment:
Shouldn't you put in a reference to Raghavender's stuff on PlanningAdaptors? It seems relevant to mention that CHOMP/STOMP can be used to post-process a path.